### DISCUSSION TITLE
The label of the segment can't support Chinese when exporting

### DISCUSSION THREAD

**Discussion by hdyihv** [mifi/lossless-cut#2223] [Q&A] [answered]
I split a video into several clips and added a label to each clip, such as "学校".  I added the "label" variable to the file name when exporting, but "学校" was displayed as "_".

**Comment by mifi** [comment, marked as answer]
https://github.com/mifi/lossless-cut/blob/master/issues.md#output-file-name-is-missing-characters-chinese-accents-etc

**Reply by hdyihv** [reply]
thanks a lot，I love losslesscut so much.
